tools/FakeItEasy.Build/Program.cs

```diff
@@ -56,6 +56,19 @@
     forEach: projectsToPack,
     action: project => Run("dotnet", $"pack {project.Path} --configuration Release --no-build --nologo --output \"{Path.GetFullPath("artifacts/output")}\""));
 
+// If mkdocs's site_url configuration is not set, sitemap.xml is full of "None" links.
+// Even with a valid site_url, the link check would fail if we added a new page.
+// So trust that sitemap.xml will be generated correctly.
+// The 404.html page is likewise generated, and isn't even the one we use on the live site (mike generates a new one).
+Target(
+    "docs",
+    DependsOn("generate-docs"),
+    () => Run("linkchecker", "--ignore-url=sitemap.xml --ignore-url=404.html --check-extern -F html/./artifacts/docs-link-check.html ./artifacts/docs/index.html"));

```

Comment:
What does the `-F` option do? Path of the report? Why `/./`?

Reply:
Format and path of the report. The `/./` is probably there because, I dunno. I got the filename via path completion in the shell and left it. Removing.

> -F TYPE[/ENCODING[/FILENAME]], --file-output TYPE[/ENCODING[/FILENAME]]
>
>    // wrap each of the following lines to match the length of the first one
>    Output to a file linkchecker-out.TYPE, $XDG_DATA_HOME/linkchecker/failures for 'failures' output,
>    or FILENAME if specified. The ENCODING specifies the output encoding, the default is that of your
>    locale. Valid encodings are listed at https://docs.python.org/library/codecs.html#standard-encodings.
>    The FILENAME and ENCODING parts of the 'none' output type will be ignored, else if the file already
>    exists, it will be overwritten. You can specify this option more than once.
>    Valid file output types are 'csv', 'xml', 'dot', 'failures', 'gml', 'gxml', 'html', 'none',
>    'sitemap', 'sql', 'text'. You can specify this option multiple times to output to more than one file.
>    Default is no file output. Note that you can suppress all console output with the option '-o none'.

Reply:
Where is `ENCODING`? You just did `TYPE/FILENAME`, is that valid? How does it know whether it's the encoding or the filename?

Reply:
It seems to be valid. I dunno. It sussed it out. I'll add an encoding.